nix/sources.nix

```diff
@@ -98,7 +98,10 @@ let
       saneName = stringAsChars (c: if isNull (builtins.match "[a-zA-Z0-9]" c) then "_" else c) name;
       ersatz = builtins.getEnv "NIV_OVERRIDE_${saneName}";
     in
-      if ersatz == "" then drv else ersatz;
+      if ersatz == "" then drv else
```

Comment:
What was the issue here? Would you like to create a ticket in https://github.com/nmattia/niv?

Reply:
Ah I should have mentioned... This was volunteered by niv itself (IIRC bumping the source version from 23 to 24).

Diff comes from invoking `niv init` and I decided to keep it for the sake of being on the latest-and-greatest version of everything :)

Reply:
hah, I see! Thanks